.github/workflows/cypress-Run-all-tests-no-cache.yml

```diff
@@ -3,17 +3,30 @@
 # This action is based on the Kitchen Sink project (https://github.com/cypress-io/cypress-example-kitchensink/)
 # It can run tests by Browser and test file selection.
 
-name: Run Cypress tests without cache
+
+name: Run Cypress tests with npm packages
 
 on: 
-  - push
-  - workflow_dispatch
+  push:
+  schedule:
+    # runs at the minute 0 of the first hour of every monday
+    - cron:  '0 1 * * 1'
```

Comment:
Use double-quotes for variables as on the rest of the document, please.

Reply:
The [official documentation examples](https://docs.github.com/es/actions/learn-github-actions/events-that-trigger-workflows#schedule) use single quotes, i believe that it's also possible to use double ones but that would go against the github actions recomendations.

Reply:
That's ok. But you are using double-quotes on the rest of the file.
Why would you mix the styles on a same file? 🤔

Reply:
You're right, solved it in the last commit